### PR TITLE
Update default return for on_pause doc

### PR DESCRIPTION
**Pull request by gfyoung**
Title is self-explanatory.  Looks like this was overlooked in #4603.

### CODE REVIEW

**Comment by gfyoung** [comment]
Added `[ci skip]` to commit because this is a simple doc change, and test failures seem to be unrelated to this PR judging from what I'm seeing on Travis and Appveyor.

File: kivy/app.py [hunk 1]
```diff
@@ -284,9 +284,8 @@ def close_settings(self, *args):
        System due to the user switching to another application, a phone
        shutdown or any other reason.
     #. :meth:`App.on_pause` is called:
-    #. If False is returned (default case), then :meth:`App.on_stop` is
-       called.
-    #. Otherwise the application will sleep until the OS resumes our App
+    #. If False is returned, then :meth:`App.on_stop` is called.
+    #. If True is returned (default case), the application will sleep until the OS resumes our App.
```
Comment (KeyWeeUsr) [inline kivy/app.py]: pep8 (<=79 chars) 😋

Comment (gfyoung) [inline kivy/app.py]: Done.

**Comment by KeyWeeUsr** [comment]
Thank you. 🍡

**Pull request merged**